src/main/java/duke/DateHandler.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Handles strings that are supposed to be dates and formats them properly.
+ * @author lukkesreysandeur
+ */
+public class DateHandler {
+    private static DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyyMMdd");
```

Comment:
Instead of having a fixed pattern inside date handler, maybe it might be preferable to have multiple types of patterns and store them in an enum class in case the user were to put in different date patterns

Reply:
Ah thats a good suggestion. I will try implementing that, thanks!